package-lock.json

```diff
@@ -20392,6 +20404,93 @@
       "integrity": "sha512-M9/ELqF6fy8FwmkpnF0S3YKOqMyoWJ4+CS5Efg2ct3oY9daQvd/Pc71FpGZsVsbl3Cpb+IIcjBDUnnyBdQbq4w==",
       "license": "MIT"
     },
+    "packages/jaeger-ui/node_modules/vite": {
+      "version": "6.0.7",
+      "resolved": "https://registry.npmjs.org/vite/-/vite-6.0.7.tgz",
```

Comment:
And somehow, renovate-bot is trying to fix that wrong lockfile by registering the module it upgrades to their own workspaces.